lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -252,18 +252,16 @@ class AwsCompileStreamEvents {
               );
             }
 
+            streamResource.Properties.BisectBatchOnFunctionError = (event.stream.bisectBatchOnFunctionError === true);
+
             if (event.stream.batchWindow) {
               streamResource.Properties.MaximumBatchingWindowInSeconds = event.stream.batchWindow;
             }
 
-            if (event.stream.maximumRetryAttempts) {
+            if (event.stream.maximumRetryAttempts >= 0) {
```

Comment:
Let's update constraint simply to `!= null` (it allow to surface eventual configuration errors)

Reply:
Thanks @medikoo , Updated this.